docs/modules/ROOT/pages/api/erc1155.adoc

```diff
@@ -0,0 +1,665 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:eip1155: https://eips.ethereum.org/EIPS/eip-1155[EIP1155]
+:receiving-tokens: xref:/erc1155.adoc#receiving_tokens[Receiving Tokens]
+:inner-src5: xref:api/introspection.adoc#ISRC5[SRC5 ID]
+
+= ERC1155
+
+Reference of interfaces, presets, and utilities related to ERC1155 contracts.
+
+TIP: For an overview of ERC1155, read our xref:erc1155.adoc[ERC1155 guide].
+
+== Core
+
+[.contract]
+[[IERC1155]]
+=== `++IERC1155++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/token/erc1155/interface.cairo[{github-icon},role=heading-link]
+
+[.hljs-theme-dark]
+```javascript
+use openzeppelin::token::erc1155::interface::IERC1155;
+```
+Interface of the IERC1155 standard as defined in {eip1155}.
+
+[.contract-index]
+.{inner-src5}
+--
+0x6114a8f75559e1b39fcba08ce02961a1aa082d9256a158dd3e64964e4b1b52
+--
+
+[.contract-index]
+.Functions
+--
+* xref:#IERC1155-balance_of[`++balance_of(account, token_id)++`]
+* xref:#IERC1155-balance_of_batch[`++balance_of_batch(accounts, token_ids)++`]
+* xref:#IERC1155-safe_transfer_from[`++safe_transfer_from(from, to, token_id, value, data)++`]
+* xref:#IERC1155-safe_batch_transfer_from[`++safe_batch_transfer_from(from, to, token_ids, values, data)++`]
+* xref:#IERC1155-is_approved_for_all[`++is_approved_for_all(owner, operator)++`]
+* xref:#IERC1155-set_approval_for_all[`++set_approval_for_all(operator, approved)++`]
+--
+
+[.contract-index]
+.Events
+--
+* xref:#IERC1155-TransferSingle[`++TransferSingle(operator, from, to, id, value)++`]
+* xref:#IERC1155-TransferBatch[`++TransferBatch(operator, from, to, ids, values)++`]
+* xref:#IERC1155-ApprovalForAll[`++ApprovalForAll(owner, operator, approved)++`]
+* xref:#IERC1155-URI[`++URI(value, id)++`]
+--
+
+==== Functions
+
+[.contract-item]
+[[IERC1155-balance_of]]
+==== `[.contract-item-name]#++balance_of++#++(account: ContractAddress, token_id: u256) → u256++` [.item-kind]#external#
+
+Returns the amount of `token_id` owned by `account`.
+
+[.contract-item]
+[[IERC1155-balance_of_batch]]
+==== `[.contract-item-name]#++balance_of_batch++#++(accounts: Span<ContractAddress>, token_ids: Span<u256>) → Span<u256>++` [.item-kind]#external#
+
+Returns a list of balances from `accounts` and `token_ids` corresponding indexes.
```

Comment:
That sounds like it should return every token balance for each account when it is just matching indexes (maybe it doesn't sound like that, I'm not the best to say it me being no a native English speaker :( ). I didn't want to go with "Batched version of balance_of" because that does not explain what the method does to someone who is not familiar with the implementation. Agree we should probably update the wording. 

Wdyt?

Reply:
Do you have a specific example? I'm sure you would come with a better phrase than me :)

Reply:
I don't know haha the more I think about it, the more I like your original description. Maybe I need to up my English game :)
Here's another way to phrase it (and let me know what you think): "Returns a list of balances derived from the `accounts` and `token_ids` pairs"

Reply:
Applied!